<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FestivalDialog</class>
 <widget class="QDialog" name="FestivalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Label" name="cTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>544</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Label" name="cSizeLabel">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>274</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="FestivalButton" name="cLargeFestivalButton">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>198</y>
     <width>430</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="FestivalButton" name="cGrandFestivalButton">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>228</y>
     <width>430</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="FestivalButton" name="cSmallFestivalButton">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>168</y>
     <width>430</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cAccept">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>269</y>
     <width>39</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cCancel">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>269</y>
     <width>39</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cReturn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>269</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Button" name="cHelp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>269</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GodButton" name="cCeresButton">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>48</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GodButton" name="cNeptuneButton">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>48</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GodButton" name="cMercuryButton">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>48</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GodButton" name="cMarsButton">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>48</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="GodButton" name="cVenusButton">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>48</y>
     <width>80</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>widget/button.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widget/label.h</header>
  </customwidget>
  <customwidget>
   <class>FestivalButton</class>
   <extends>QPushButton</extends>
   <header>dialog/festivaldialog.h</header>
  </customwidget>
  <customwidget>
   <class>GodButton</class>
   <extends>QPushButton</extends>
   <header>dialog/festivaldialog.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
